src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String indentation = "    ";
+    public static String doubleIndentation = indentation + "  ";
+    public static String tripleIndentation = doubleIndentation + "  ";
+    public static String horizontalLine = indentation
+            + "<------------------------------------------------------------>\n";
+    public static Task[] Tasks = new Task[100];
```

Comment:
I think it would make the code more readable if you replace the 100 with a constant name, maybe "MAX_TASKS" to describe the number?

Also, I think Tasks is a variable - should it be named "tasks" instead? :)

Reply:
Oh, thank you! I didn't notice that. :D